src/idsk/components/footer-extended/footer-extended.scss

```diff
@@ -96,80 +81,313 @@
 
             &:hover,
             &:active {
-                color: #0b0c0c;
+                color: govuk-colour("black");
             }
         } @else {
             &:link,
             &:visited,
             &:hover,
             &:active {
-                color: #0b0c0c;
+                color: govuk-colour("black");
             }
         }
     }
 
-    .bordering {
-        border-top: 1px solid #bfc1c3;
+    .idsk-description-panel {
+        border-top: 1px solid govuk-colour("mid-grey");
         padding-top: 7px;
     }
 
-    .one-third {
-        padding-top: 28px;
-    }
-
+    .one-third,
     .two-thirds {
         padding-top: 28px;
     }
 
-    .small_subtitle {
-        border-top: 1px solid #bfc1c3;
+    .idsk-subtitle {
+        border-top: 1px solid govuk-colour("mid-grey");
         padding-top: 7px;
     }
 
-    .idsk-footer__meta-item {
+    .idsk-footer-extended-meta {
+        display: -webkit-box;
+        display: -ms-flexbox;
+        display: flex; // Support: Flexbox
+        margin-right: -$govuk-gutter-half;
+        margin-left: -$govuk-gutter-half;
+        -ms-flex-wrap: wrap;
+            flex-wrap: wrap; // Support: Flexbox
+        -webkit-box-align: start;
+            -ms-flex-align: start;
+                align-items: flex-start; // Support: Flexbox
+        -webkit-box-pack: center;
+            -ms-flex-pack: center;
+                justify-content: center; // Support: Flexbox
+      }
+
+    .idsk-footer-extended-meta-item {
         margin-right: 15px;
         margin-bottom: 25px;
-        margin-left: 0px;
+        margin-left: 0;
         padding-bottom: 20px;
     }
 
-    .logo {
+    .idsk-footer-extended-meta-item--grow {
+        flex: 1; // Support: Flexbox
+        @include mq ($until: tablet) {
+          flex-basis: 320px; // Support: Flexbox
+        }
+      }
+
+    .idsk-ministri-logo {
         flex: none;
         order: 1;
         align-self: flex-end;
         flex-grow: 0;
         padding-top: 8px;
     }
 
-    .frame {
+    .idsk-frame {
         display: inline-block;
         width: 100%;
         height: 30px;
-        left: 0px;
-        top: 0px;
-
+        left: 0;
+        top: 0;
         font-style: normal;
         font-weight: 400;
         font-size: 17px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
     }
 
-    .label {
+    .idsk-label {
         font-style: normal;
         font-weight: normal;
         font-size: 19px;
         line-height: 25px;
-
-        color: #0b0c0c;
+        color: govuk-colour("black");
         text-decoration: underline;
     }
 
-    .bottom-space {
+    .idsk-grid-column {
```

Comment:
taktiez naming.. a vlastne aby som to nepisal vsade tak podoplnaj radsej ten prefix z idsk- na idsk-footer-extended